src/libraries/periphery/MorphoBalancesLib.sol

```diff
@@ -105,8 +105,10 @@ library MorphoBalancesLib {
     }
 
     /// @notice Returns the expected borrow assets balance of `user` on a market after having accrued interest.
-    /// @dev Warning: repaying a borrow position using the expected assets balance can lead to a revert due to
-    /// conversion roundings between shares and assets.
+    /// @dev Warning: The expected balance is rounded up, so it can be greater than the market's expected total supply
+    /// assets.
+    /// @dev Warning: Repaying using the expected supply assets can lead to a revert due to conversion roundings from
+    /// assets to shares.
```

Comment:
Do we have a concrete example of that ? It may not be an issue because of the zeroFloorSub/min in repay/liquidate

Reply:
Oh, that's right, so maybe this is what the comment was about, but badly phrased